docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -27,7 +27,12 @@
       when: deployment_config is defined
 
     - set_fact:
-        manager_app_chart: "{{ manager_app_chart | default('/usr/local/share/applications/helm/deployment-manager-*.tgz') }}"
+        manager_app_chart: "{{ manager_app_chart | default('wind-river-cloud-platform-deployment-manager.tgz') }}"
```

Comment:
After changing the default value here, is the 'wind-river-cloud-platform-deployment-manager.tgz' able to be uploaded?

Reply:
Updated with the proper default value